mep/people/views.py

```diff
@@ -63,33 +63,42 @@ def get_form(self, *args, **kwargs):
         # initialize the form, caching on current instance
         if not self._form:
             self._form = super().get_form(*args, **kwargs)
-        # set minimum and maximum years for date range field
-            min_max = self.get_year_range()
-            if min_max:
-                self._form.set_membership_dates_placeholder(*min_max)
+        # set minimum and maximum years for date range fields
+            min_max_ranges = self.get_year_ranges('birth_year',
+                                                  'account_years')
+            if min_max_ranges:
+                self._form.set_daterange_placeholders(min_max_ranges)
         # Get facets from solr return
         return self._form
 
     @staticmethod
-    def get_year_range():
+    def get_year_ranges(*field_names):
         """Return the earliest and latest years for any account activity in
         the library.
 
-        :return: Min and max years as integers or None
-        :rtype: tuple or None
+        :param \*field_names:
+            Field names for which to return min and max years.
+        :type \*field_names: tuple
+
+        :returns: Field-keyed min and max years as integers
```

Comment:
This description is not clear to me when I read it. How about:
> dictionary keyed on field name with a tuple of (min, max) as integer

Probably worth noting what happens if stats are not returned from Solr for a field - key is not added to the dict, right?

Reply:
Changed on both counts.